capivaraProjects/src/views/addUser.tsx

```diff
@@ -175,6 +226,27 @@ export class addUser extends React.Component<{},AddUserState>{
 
     }
 
+    private async createUserInServer(newUser : User) : Promise<void>{
+
+        try{
+            
+            this.setState({ loading : true} );
+            await createUser(newUser);
+            Alert.alert('Usuário criado com sucesso!');
+            
+        }catch(error){
```

Comment:
Ao criar um usuário com sucesso, ele poderia retornar pra listagem. O que acha?

Reply:
Não entendi muito bem alan, tipo eu imagino q se criei ele. Quando eu for dar loading eu devo encontrar ele certo?

Reply:
Sua aplicação poderia retornar pra página da lista de usuári após criar um.